src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -36,6 +35,7 @@ export const NEW_ADDRESS = "NEW_ADDRESS"
 const PAGE_SIZE = 30
 
 interface SavedAddressesProps {
+  handeChangesAddressCount?: (active?: number) => void
```

Comment:
The general idiomatic rule of thumb with callbacks in React is that callbacks passed down as props use the "onSomething" naming form, and callbacks that are defined internal to the component use the `handleSomething` form.